File: Java/Java-Kompendium  - Professionell Java programmieren lernen/uebung2a.java
/*
1.  Erstellen Sie ein Programm, das drei Variablen für ganze Zahlen enthält.
    Diese sollen jedoch jeweils einen unterschiedlichen Typ verwenden.
    Geben sie einer von ihnen einen beliebigen Wert.
    Weisen Sie daraufhin den beiden anderen Variablen den Wert der ersten Variablen zu.
    Wählen Sie dafür Typen, die keine explizite Umformung des Variablentyps notwendig machen.

Neumann, Markus. Java: Kompendium: Professionell Java programmieren lernen (German Edition) (S.60-61). Kindle-Version.
*/

public class uebung2a
{
    public static void main(String[] args)
    {
        byte x = 1;
        int y = 2;
        long z = 3;

        System.out.println("Werte: x: " + x + " y: " + y + " z: " + z);
        
        y = x;
        z = x;

        System.out.println("Werte: x: " + x + " y: " + y + " z: " + z);
    }
}

File: C/C - Programmieren für Einsteiger/uebung2a.c
/*
1.  Schreiben Sie ein Programm, das eine Variable für eine ganze Zahl definiert und ihr einen beliebigen Wert zuweist.
    Deklarieren Sie daraufhin zwei neue Variablen und weisen der ersten den doppelten Wert und der zweiten das Quadrat der ursprünglichen Variablen zu.
    Geben Sie daraufhin die entsprechenden Ergebnisse aus.

Neumann, Markus. C Programmieren: für Einsteiger: Der leichte Weg zum C-Experten (Einfach Programmieren lernen 8) (German Edition) (S.55). Kindle-Version.
*/

#include <stdio.h>

void main()
{
    int x = 2;
    int y = x + x;
    int z = x * x;

    printf("Doppelt: %d \n", y);
    printf("Quadrat: %d \n", z);
}